Documentation/compatibility/aspnet-Invalid results from HttpRequest.GetAttributeFromHeader.md

```diff
@@ -0,0 +1,37 @@
+## ASP.NET System.Web.HttpRequest.GetAttributeFromHeader can return invalid results
```

Comment:
A problem, though, is that GetAttributeFromHeader is a private method. It would be useful to list some of the calling methods in the public surface area.

Reply:
see the latest update